models/models.go

```diff
@@ -73,15 +73,34 @@ type UserData struct {
 	Key   []byte
 }
 
-func CreateUser(email string) (utils.User, error) {
-	user := utils.User{Email: email}
+/*
+ * LoadRandomUser The Let's Encrypt v1 acme API has shut down user creation to force users to adopt v2.
+ * In an attempt to contiune using v1 while we develop a v2 compliant broker, we are replacing
+ * calls to create a new user for each new domain registration with a method that fetches an existing user
+ * from a pool of ids. The random selection of users from a pool aims to minimize the impact of the following rate limits:
+ *	- 300 Pending Authorizations per account
+ *	- Failed Validation limit of 5 failures per account, per hostname, per hour.
+ */
+func LoadRandomUser(db *gorm.DB, userIDPool []string) (utils.User, error) {
+	var user utils.User
+	userID := userIDPool[rand.Intn(len(userIDPool))]
```

Comment:
```suggestion
        defer func() { if r: = recover(); r != nil { return }}()
	userID := userIDPool[rand.Intn(len(userIDPool))]
```

`math.rand.RandN()` will sometimes panic if the number is zero. I have no idea why, but I've been bitten by this in the past, so just add the deferred recovery to avoid dying (and pick a default).

Source: https://golang.org/pkg/math/rand/#Rand.Intn